agent/ebs/watcher_windows.go

```diff
@@ -0,0 +1,21 @@
+//go:build windows
+// +build windows
+
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//      http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package ebs
+
+const (
+	hostMountDir = "C:\\var\\lib\\kubelet"
```

Comment:
Q: Do we also need a `csiDriverSocketAddress` here for windows?

Reply:
We load the socket path from the default Windows config in `\agent\config\config_windows.go`